### ISSUE TITLE
Missing Alt Text on image gallery

### ISSUE THREAD

**Issue by RiaanKnoetze** [woocommerce/woocommerce#15564]
When looking at HTML markup for images in the gallery on the single product page, user noticed that alt text are no longer showing up.

Compare the following screenshots:

## Frontend

![http://cld.wthms.co/arLCb](http://cld.wthms.co/arLCb+)
Full Image: http://cld.wthms.co/arLCb+

## Media setup

![http://cld.wthms.co/HmFnQ](http://cld.wthms.co/HmFnQ+)
Full Image: http://cld.wthms.co/HmFnQ+

**Comment by mikejolley** [comment]
This is simply the view the slider script generates for the images. The actual images (still in the markup), as well as the gallery when the slider is disabled, has alt text and captions and so on:

https://dl.dropboxusercontent.com/s/ub3q5owzzyyvxb3/2017-06-12%20at%2011.30.png
